cvat/apps/engine/tests/test_rest_api.py

```diff
@@ -30,16 +30,16 @@
 
 from datumaro.util.test_utils import TestDir
 from cvat.apps.engine.models import (AttributeSpec, AttributeType, Data, Job, Project,
-    Segment, StatusChoice, Task, Label, StorageMethodChoice, StorageChoice, DimensionType,
-    SortingMethod)
+    Segment, StageChoice, StatusChoice, Task, Label, StorageMethodChoice, StorageChoice)
 from cvat.apps.engine.media_extractors import ValidateDimension, sort
+from cvat.apps.engine.models import DimensionType, SortingMethod
```

Comment:
Why we can't have one import `from cvat.apps.engine.models ...`?

Reply:
I have fixed that. We can. It was an artifact after merging.